src/app/app-routing.module.ts

```diff
@@ -15,6 +16,7 @@ const routes: Routes = [
     canActivate: [AuthGuard]  
   },
   { path: 'login', component: LoginComponent },
+  { path: 'password/reset', component: PasswordResetComponent},
```

Comment:
```suggestion
  { path: 'reset-password', component: PasswordResetComponent},
```
Would prefer to keep it as a single path without extra slashes.